test/e2e/slow_test.go

```diff
@@ -20,14 +20,14 @@ func TestBasicThenNoRBACThenRBAC(t *testing.T) {
 	basicShareSetupAndVerification(testArgs)
 
 	framework.DeleteShareRelatedRBAC(testArgs)
-	t.Logf("%s: wait up to 10 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testArgs.Name)
+	t.Logf("%s: wait up to 11 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testArgs.Name)
 	testArgs.SearchStringMissing = true
-	testArgs.TestDuration = 10 * time.Minute
+	testArgs.TestDuration = 11 * time.Minute
 	testArgs.SearchString = "invoker"
 	framework.ExecPod(testArgs)
 
 	framework.CreateShareRelatedRBAC(testArgs)
-	t.Logf("%s: wait up to 10 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testArgs.Name)
+	t.Logf("%s: wait up to 11 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testArgs.Name)
```

Comment:
Does this need the `testArgs.TestDuration` or will it roll over from the previous one?

Reply:
rolls over from previous

Reply:
Ok, thanks.